packages/ckeditor5-ui/src/panel/balloon/balloonpanelview.ts

```diff
@@ -899,9 +903,9 @@ export function generatePositions( options: {
 	config?: object;
 } = {} ): Record<string, PositioningFunction> {
 	const {
-		sideOffset = BalloonPanelView.arrowSideOffset,
-		heightOffset = BalloonPanelView.arrowHeightOffset,
-		stickyVerticalOffset = BalloonPanelView.stickyVerticalOffset,
+		sideOffset = ARROW_SIDE_OFFSET,
+		heightOffset = ARROW_HEIGHT_OFFSET,
+		stickyVerticalOffset = STICKY_VERTICAL_OFFSET,
```

Comment:
This way you disable the option to override those offsets (it was possible to override values of public static props, but now it is not).

Reply:
Fixed in https://github.com/ckeditor/ckeditor5/commit/a2279f53ec7455329e41fc80ce2f41cd7632bc59